clients/admin-ui/src/features/common/nav/v2/nav-config.test.ts

```diff
@@ -80,7 +81,7 @@ describe("configureNavGroups", () => {
       title: "Data map",
       children: [
         { title: "View map", path: "/datamap" },
-        { title: "View systems", path: "/system" },
+        { title: "View systems", path: "/systems" },
```

Comment:
Is there a specific reason why only line 51 is using the `SYSTEM_ROUTE` constant?